wiki/Help_centre/Beatmapping/it.md

```diff
@@ -0,0 +1,51 @@
+---
+tags: 
+  - upload
+  - delete
+  - beatmap
+  - slot
+  - editor
+  - status
+  - warning
+  - caricare
+  - attenzione
+  - 
+---
+
+# Beatmapping e l'Editor
+
+*Pagina principale: [Centro di Supporto](/wiki/it/Help_centre)*
+
+Questa sezione è dedicata ai problemi riguardo il Beatmap Editor e generalmente beatmapping.  
+
+## Come carico/aggiorno la mia mappa? {#upload-beatmap}
+
+**Puoi farlo tramite l'editor in gioco.**
+
+Per caricare la tua beatmap, vai nell'editor e clicca `File` > `Carica Beatmap`.
+
+Questo renderà la tua beatmap disponibile da scaricare agli altri giocatori sul sito web. La stessa procedura vale nel caso si voglia aggiornare una mappa già caricata.
+
+Ogni utente ha un numero limitato di mappe che possono caricare. Lasciare la mappa inattiva per un mese (28 giorni), senza aggiornarla, verrà mandata nelle [Beatmap Abbandonate](/wiki/Beatmap/Category#graveyard). Per risvegliare una beatmap da quelle abbandonate a quelle in attesa o WIP, basta aggiornarla.
+
+## l'editor mi dice che "la beatmap che stai cercando di caricare non è tua!" {#wrong-author}
+
+**Questo può succedere nel caso il creatore della beatmap non è propriamente impostato nei file della beatmap.**
+
+Nell'editor, clicca `File` nel menu, poi `Apri .osu in Notepad`. Controlla la riga `Creator:` sotto la sezione `Metadata` e assicurati che combaci esattamente con il tuo nome utente, Maiuscole e minuscole incluse.
+
+Nel caso non lo fosse, cambia manualmente la riga per ogni difficoltà nella tua beatmap, ricarica la mappa e dovresti essere in grado di caricarla.
+
+Se hai ancora problemi o desideri avere più informazioni a riguardo, ti invitiamo a leggere la [Pagina riguardante problemi BSS](/wiki/Guides/BSS_Issues).
+
+## Sono un osu!supporter, ma non ho 20 slot per le mappe in attesa! {#upload-slots}
+
+**Il numero di slot disponibili dipende dal tuo numero di mappe classificate. Per maggiori dettagli, leggi [osu!supporter § Limiti aumentati](/wiki/osu!supporter#increased-limits).**
```

Comment:
Same as before, the italian page for osu! supporter is outdated, we might want to fix this in a future PR